src/convertToObject.js

```diff
@@ -6,7 +6,36 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const sourceStyle = sourceString.split(';');
+  const style = {};
+
+  Object.create(style);
+
+  sourceStyle.forEach((source) => {
+    const data = source.split(':');
+
+    if (data[0].trim()) {
+      const key = data[0].trim();
+      const value = data[1];
```

Comment:
You should trim the `value` here as well to ensure there are no leading or trailing spaces in the CSS value.